src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -396,7 +396,25 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
 
   private async _copyYaml() {
```

Comment:
we should use the util function: <https://github.com/home-assistant/frontend/blob/dev/src/common/util/copy-clipboard.ts#L1>